webrender_traits/src/channel_mpsc.rs

```diff
@@ -3,7 +3,6 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 use serde::{Deserialize, Serialize};
-use std::io::Error;
```

Comment:
We don't use io::Error at all here. we could return std::sync::mpsc::RecvError, but since the channel_ipc version returns another type of error, we won't be able to poke at the error value anyway (other than print it to stdout).